examples/water_boiler/water_boiler.py

```diff
@@ -44,15 +63,19 @@ def update(self, boiler_power, dt):
         power = pid(water_temp)
         water_temp = boiler.update(power, dt)
 
-        x += [current_time - start_time]
-        y += [water_temp]
+        x += [round(current_time - start_time, 3)]
+        y += [round(water_temp, 2)]
```

Comment:
Is there a specific reason you decided to round these values? I'm not questioning it necessarily, I'm just curios why you decided to do it. Was there a problem you noticed without the rounding? And why did you round one value to 3 digits and the other value to 2?

Reply:
personally, it is a common format for time in ms and temperature in 0.01.